providers/discord/src/airflow/providers/discord/notifications/discord.py

```diff
@@ -22,7 +22,7 @@
 from airflow.providers.common.compat.notifier import BaseNotifier
 from airflow.providers.discord.hooks.discord_webhook import DiscordWebhookHook
 
-ICON_URL: str = "https://raw.githubusercontent.com/apache/airflow/main/airflow/www/static/pin_100.png"
+ICON_URL: str = "https://raw.githubusercontent.com/apache/airflow/2.5.0/airflow/www/static/pin_100.png"
```

Comment:
It should be
`https://raw.githubusercontent.com/apache/airflow/main/airflow-core/src/airflow/ui/public/pin_32.png`
though I am not sure about the size impact

@bbovenzi do you know why we removed the pin_100.png from source code?

Reply:
Probably a mistake when removing FAB.

Reply:
Please do!

Reply:
Fixed in https://github.com/apache/airflow/pull/48725